docs/release-notes/v13.0.1.md

```diff
@@ -0,0 +1,118 @@
+
+### Improvements
+
+### Improved Console navigation with `ctrl/cmd + k`
+
+The navigation with `ctrl/cmd + k` has been revised and improved for a better user experience when navigating among resources. Users can now easily see which Companies and Projects they belong to and can quickly search within these resources and seamlessly jump to them.
+
+### Interpolation in Annotations and Labels for Marketplace items
+
+When defining a new service on the Markeplace, it is now possible to configure the desired interpolations for Annotations and Labels in it, so that when the respective resource is created on a given Project on Console, the interpolated values are correctly set.
+
+### Delete a Microservice in `Advanced` tab
+
+The experience of deleting a Microservice has been revised: from now on, if you want to delete a Microservice, you need to do so within the `Advanced` tab in the Microservice detail page.
+
+### Raw Manifest Configuration in `Advanced` tab
+
+Configuring the raw manifest of a Microservice is now possible within the `Advanced` tab in the Microservice detail page.
+
+### Git repository name validation accepts aldo dots
+
+In the creation of a template or an example, you can now insert dots inside the Git Repository Name, as the validation now allows such inclusion.
+
+### Human readable error validations in Create new branch modal & Project creation page
+
+Error validations when trying to create a new Branch or a new Project appear now more human readable, improving clarity and user understanding.
+
+### Enable Microservice metric gathering in a `Monitoring` tab
+
+From now on, the Microservice monitoring can be managed in the `Monitoring` tab of each microservice.
+
+### Bug Fix
+
+This version addressed the following bugs:
+
+* fixed a bug that wrongly flashed the Create Project button when entering a Company without the Company Owner role;
+* fixed the flicker of nodes and edges on configuration or selection changes in Flow Manager section;
+* fixed a bug that caused an error on configuration save when Console version is increased from v12 to v13 and the Flow Manager Configurator is not opened;
+* fixed a bug that caused a malfunction in reading environment variables in the 'variables.env' file when creating and first configuring a new environment in console;
+* fixed bug in project creation that prevented the user from finding the default template set by Blueprint section;
+* fixed bug that caused a problem with redirects in Runtime area;
+* fixed bug that prevented saving configuration on projects with Azure as provider, if the corresponding project was empty.
+
+## Fast Data
+
+### Fast Data Configurator
+
+#### Generate from sample functionality in Single View section
+
+Added the "generate from sample" functionality also to Single View section allowing users to generate Single Views starting from a JSON or CSV data sample

```

Comment:
```suggestion
#### Generate Single view fields from data sample

From the `Import fields` button, an option has been added that allows you to generate fields ** From a data sample**. 
Doing this will replace the current fields with those contained within the file.
The supported file extensions are: `.csv` and `.json`.
```